pyuoi/linear_model/base.py

```diff
@@ -291,7 +291,8 @@ def fit(self, X, y, stratify=None, verbose=False):
                     y_pred = self.estimation_lm.predict(X_test[:, support])
                 else:
                     # no prediction since nothing was selected
-                    y_pred = np.zeros(y_test.size)
+                    y_pred = self.selection_lm.predict(
```

Comment:
I'm not sure that this will work for all models. For Lasso/OLS, there is a closed form solution for the intercept, which sets the predictions with no features. But, for mutinomial-logistic or poisson, I think the value of the intercept depends on the beta weights/features, so I think just using the last selection_lm will give results that depend on what the last bootstrap was.

I think one solution that should work for all models is to do something like
```python
y_pred = self.selection_lm.fit(np.zeros_like(X_rep), y_rep).predict(np.zeros_like(X_test))
```

In this case, the prediction model only depends on y_rep (train) and not on any features.

Reply:
Actually, I think this would work for poisson, but I'm not sure it does for multinomial-logistic.